packages/lib/src/vis/shared/VisCanvas.tsx

```diff
@@ -81,15 +81,14 @@ function VisCanvas(props: PropsWithChildren<Props>) {
                 {children}
                 <ViewportCenterer />
                 <RatioEnforcer visRatio={visRatio} />
-                <Html>
-                  <div
-                    ref={(elem) => setFloatingToolbar(elem || undefined)}
-                    className={styles.floatingToolbar}
-                  />
-                </Html>
               </InteractionsProvider>
             </AxisSystemProvider>
           </Canvas>
+
+          <div
+            ref={(elem) => setFloatingToolbar(elem || undefined)}
+            className={styles.floatingToolbar}
+          />
```

Comment:
Now that `canvasWrapper` has `position: relative`, I can actually move the floating toolbar directly inside it to avoid rendering an `Html` element.